www/pfSense-pkg-squid/files/usr/local/www/squid_clwarn.php

```diff
@@ -19,7 +19,9 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-$VERSION = '6.10';
+require_once("pkg-utils.inc");
+pkg_exec("query '%v' squidclamav", $version, $err);
```

Comment:
pkg_exec returns 0 for success and != 0 when it fails. It would be good to check if return is 0 before assume $version really contains a valid value